frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -350,12 +344,13 @@ class DashboardHeader extends Component {
               }
             />
           </Tooltip>,
-          <EntityMenu
-            key="dashboard-action-menu-button"
-            items={extraButtons}
-            triggerIcon="ellipsis"
-            tooltip={t`Move, archive, and more...`}
-          />,
+          <DashboardHeaderButtonContainer key="dashboard-action-menu-button">
```

Comment:
Some buttons are wrapped with a container since it's not possible to style the button itself due to their implementations and usages.